apps/web/cypress/tests/organization-switch.spec.ts

```diff
@@ -20,7 +20,7 @@ describe('Organization Switch', function () {
 
       cy.getByTestId('organization-switch').focus();
 
-      cy.get('.mantine-Select-item').contains(capitalize(newOrg.name)).click();
+      cy.get('.mantine-Select-item').contains(capitalize(newOrg.name)).click({ force: true });
```

Comment:
this e2e test was failing to me

Reply:
Is there a way to not need force here?

Reply:
I haven't checked but saw a couple of times that we do force in other places. This test was failing to me locally but I'm wondering how it's passing on the next branch.